assets/css/_crowding.scss

```diff
@@ -64,22 +64,6 @@ $color-crowding-icon-bg: $color-bg-medium;
 .m-crowding-diagram__status-description {
   font-weight: 500;
   text-transform: uppercase;
-
-  &--empty {
-    color: $color-crowding-empty-text;
```

Comment:
nitpick(non-blocking): Is the `$color-crowding-empty-text` variable still needed?

Reply:
Nice catch! seems like it's not, but I could actually leave the `--empty` selector and keep styling the empty text the same.

That's sorta underspecified in the ticket.